webapp-django/crashstats/signature/views.py

```diff
@@ -518,17 +497,17 @@ def _transform_graphics_summary(facets):
                     'adapter': adapter['term'],
                     'count': adapter['count'],
                 }
-                key = '{}:{}'.format(vendor['term'], adapter['term'])
-                names = hexes.get(key)
+                key = (adapter['term'], vendor['term'])
+                names = all_names.get(key)
                 if names and names[0]:
-                    entry['vendor'] = '{} ({})'.format(
+                    entry['adapter'] = '{} ({})'.format(
                         names[0],
-                        vendor['term'],
+                        adapter['term'],
                     )
                 if names and names[1]:
-                    entry['adapter'] = '{} ({})'.format(
+                    entry['vendor'] = '{} ({})'.format(
                         names[1],
-                        adapter['term'],
+                        vendor['term'],
```

Comment:
Why did you revert ``adapter`` and ``vendor``? If you can avoid it, the diff would be quite smaller.